### ISSUE TITLE
Selections by property ( ALA and beta < 0.7)

### ISSUE THREAD

**Issue by sperezconesa** [nglviewer/ngl#795]
Hello,

I think it would a great feature for NGL if selection could be done based on a property like:
```python
'ALA and beta<0.8'
'within 3 of ALA'
```
It doesn't even have to update dynamically. I imagine it has probably been suggested already but I don't know if there is any plans of implementation. 

Thank you very much and keep up the good work,
Sergio

**Comment by fredludlow** [comment]
Yes! Both property and distance expressions would be great to have.
